images/base/files/etc/containerd/config.toml

```diff
@@ -17,4 +17,6 @@ version = 2
   sandbox_image = "k8s.gcr.io/pause:3.3"
   # allow hugepages controller to be missing
   # see https://github.com/containerd/cri/pull/1501
-  tolerate_missing_hugepages_controller = true
\ No newline at end of file
+  tolerate_missing_hugepages_controller = true
+  # explicitly use default snapshotter so we can sed it in entrypoint
+  snapshotter = "overlayfs"
```

Comment:
it says that depends on the platform, is this something we worry about?
https://godoc.org/github.com/containerd/containerd#pkg-constants

Reply:
nevermind, this will be always linux